src/coreclr/pal/src/synchmgr/wait.cpp

```diff
@@ -229,6 +229,7 @@ SignalObjectAndWait(
         bAlertable ? "TRUE" : "FALSE");
 
     CPalThread *thread = InternalGetCurrentThread();
+
```

Comment:
A nit - there is a couple of files with the only change being an added empty line. Can you please remove them?